cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -365,6 +367,15 @@
     </description>
   </property>
 
+  <property>
+    <name>program.container.dist.jars</name>
+    <value></value>
+    <description>
+      Extra jars to be localized in every Twill container and to be
```

Comment:
They are localized in `AbstractDistributedProgramRunner`, so probably not for CDAP system services?